src/shell/subproc.zig

```diff
@@ -1146,74 +1181,28 @@ pub const PipeReader = struct {
-                    }
-                }
-
-                this.captured_writer.writer.outgoing.write(chunk) catch bun.outOfMemory();
-            } else if (this.captured_writer.writer.getPoll() == null) {
-                if (this.captured_writer.writer.start(writer_fd, true).asErr()) |e| {
-                    _ = e; // autofix
-                    Output.panic("TODO SHELL SUBPROC onReadChunk error", .{});
-                }
-            }
-
-            // if (this.captured_writer.writer.start(writer_fd, true).asErr()) |e| {
-            //     const writer = std.io.getStdOut().writer();
-            //     e.format("Yoops ", .{}, writer) catch @panic("oops");
-            //     @panic("TODO SHELL SUBPROC onReadChunk error");
-            // }
-
-            // if (comptime Environment.isWindows) {
-            //     if (this.captured_writer.writer.source == null) {
-            //         if (this.captured_writer.writer.start(writer_fd, true).asErr()) |e| {
-            //             const writer = std.io.getStdOut().writer();
-            //             e.format("Yoops ", .{}, writer) catch @panic("oops");
-            //             @panic("TODO SHELL SUBPROC onReadChunk error");
-            //         }
-            //     }
-            // } else {
-            //     if (this.captured_writer.writer.getPoll() == null) {
-            //         this.captured_writer.writer.handle = .{ .poll = Async.FilePoll.init(this.eventLoop(), writer_fd, .{}, @TypeOf(this.captured_writer.writer), &this.captured_writer.writer) };
-            //     }
-            // }
-
-            log("CapturedWriter(0x{x}, {s}) write", .{ @intFromPtr(&this.captured_writer), @tagName(this.out_type) });
-            if (bun.Environment.isWindows) {
-                _ = this.captured_writer.writer.flush();
-            } else switch (this.captured_writer.writer.write(chunk)) {
-                .err => |e| {
-                    const writer = std.io.getStdOut().writer();
-                    e.format("Yoops ", .{}, writer) catch @panic("oops");
-                    @panic("TODO SHELL SUBPROC onReadChunk error");
-                },
-                else => |result| {
-                    log("CapturedWriter(0x{x}, {s}) write result={any}", .{ @intFromPtr(&this.captured_writer), @tagName(this.out_type), result });
-                },
-            }
-        }
+        this.captured_writer.doWrite(chunk);
 
         const should_continue = has_more != .eof;
 
         if (should_continue) {
             if (bun.Environment.isPosix) this.reader.registerPoll() else switch (this.reader.startWithCurrentPipe()) {
                 .err => |e| {
-                    const writer = std.io.getStdOut().writer();
-                    e.format("Yoops ", .{}, writer) catch @panic("oops");
+                    const writer = std.io.getStdErr().writer();
+                    e.format("Yoopsy ", .{}, writer) catch @panic("oops");
```

Comment:
What is the Yoopsy oops about? The panic should have a user friendly message to report.